SU-HSDO/stanford_mrc
# Mrcd8 156

# READY FOR REVIEW

# Summary
- Front-end code and markup for News Card component
- Refactored postcard and added mixins

# Needed By (Date)
- End of sprint (12/6)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify markup file in browser (adding Decanter/Stanford Basic stylesheet on top gives you true look and feel)

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-156

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
